.travis.yml

```diff
@@ -0,0 +1,88 @@
+language: php
+
+matrix:
+  include:
+    - php: 5.5
+      env: TEST_NAME=oc-tests/test_reshareDir
+    - php: 5.5
+      env: TEST_NAME=oc-tests/test_scrapeLogFile
+    - php: 5.5
+      env: TEST_NAME=oc-tests/test_shareDir
+    - php: 5.5
+      env: TEST_NAME=oc-tests/test_shareFile
+    - php: 5.5
+      env: TEST_NAME=oc-tests/test_shareGroup
+    - php: 5.5
+      env: TEST_NAME=oc-tests/test_sharePermissions
+    - php: 5.5
+      env: TEST_NAME=oc-tests/test_uploadFiles
+  allow_failures:
+    - env: TEST_NAME=oc-tests/test_shareGroup
+    - env: TEST_NAME=oc-tests/test_sharePermissions
+  fast_finish: true
+
+branches:
+  only:
+    - master
+    - /^stable\d*$/
+
+env:
+  global:
+    - CORE_BRANCH=stable8
```

Comment:
so the intention  here is to test smashbox or owncloud by running smashbox?

From my understanding the first - master or latest stable